.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Thank you for sending the PR! We appreciate you spending the time to work on these changes.
```

Comment:
I wonder if we need this much happy talk. The text box is only 15 lines tall on my computer, and that means that the `Description` header is the bottom line visible without scrolling. Honestly I would drop this entire comment and just start with "Have you read ..."